### PR TITLE
Added cake.config download command

### PR DESCRIPTION
**Pull request by gep13**
Relates to #10

### CODE REVIEW

**Comment by gep13** [comment]
@patriksvensson very much a work in progress.  It "seems" to work, but literally just thrown it together, so there might still be some issues with it.  Let me know what you think.

File: package.json [hunk 1]
```diff
@@ -80,7 +85,12 @@
 						"linux": "http://cakebuild.net/download/bootstrapper/linux"
 					},
 					"description": "The Cake bootstrapper URIs."
-				}
+				}, "cake.configuration": {
```
Comment (patriksvensson) [inline package.json]: Seems to be some mixed spaces and tabs in this file.

File: src/configuration/cakeConfiguration.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var request = require('request');
+import vscode = require('vscode');
+import * as path from 'path';
+
+export class CakeConfiguration {
+
+    constructor() {
```
Comment (patriksvensson) [inline src/configuration/cakeConfiguration.ts]: I think this constructor can be removed.

File: src/configuration/cakeConfigurationCommand.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  if (workspace.rootPath === undefined) {
+      window.showErrorMessage('You have not yet opened a folder.');
+      return;
+  }
+
+  // Create the configuration object
+  let configuration = new CakeConfiguration();
+
+  // Does the configuration already exist?
+  var buildFilePath = configuration.getTargetPath();
```
Comment (patriksvensson) [inline src/configuration/cakeConfigurationCommand.ts]: Maybe we should rename this to `targetPath` or something.

**Comment by patriksvensson** [comment]
@gep13 I'm far from an expert on this, but it looks good to me. Left some comments.

**Comment by gep13** [comment]
@patriksvensson I have fixed up all the comments that you made, and I think this is now ready for merging.

**Comment by gep13** [comment]
@patriksvensson thanks for merging!  Can you let me know if/when you push out a new version of the extension? Thanks!

**Comment by patriksvensson** [comment]
Awesome!

**Pull request merged**